src/MessageId.h

```diff
@@ -41,6 +41,7 @@ class MessageId : public Napi::ObjectWrap<MessageId> {
  private:
   static Napi::FunctionReference constructor;
   pulsar_message_id_t *cMessageId;
+  bool skipcMessageIdFree = false;
```

Comment:
I think skip**C**MessageIdFree is correct as a camel case variable.

Reply:
Fixed it.